### PR TITLE
Availability

### PR DESCRIPTION
**Pull request by Mafusn**
### Availability enhancing

- Cleanup in routes and added a new route called /login
- Added new page to verify the given key by officials at the polling stations
- Bigger text in login screen
- Possible to enlarge the text in the voting page
- Possible to login and vote using only the keyboard (using Tab)

### Fixes

- Possible to vote blank
- Website name is changed from "Vite + React" to "Election in DK"

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,22 +4,20 @@
...
+    "dev": "npm-run-all --parallel dev:server dev:lint",
+    "dev:server": "vite",
+    "dev:lint": "nodemon --watch src --exec \"npx eslint .\"",
     "build": "vite build",
     "lint": "eslint .",
     "test": "NODE_OPTIONS='--experimental-vm-modules' jest --passWithNoTests",
-    "preview": "vite preview",
-    "cy:run": "cypress run",
-    "cy:open": "cypress open --e2e --browser chrome --port 52750",
-    "cy:server": "vite --port 2525"
```
Comment (Jatewo) [inline package.json:14]: no removing cypress 👎

File: src/Screens/VerifyKeyPage.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            // Display error if verification fails
+            setError('Forkert nøgle, prøv venligst igen.');
+        }
+    };
+
+    return (
+      <>
+        <div className='verify-key-page-div'>
+            <h1>Indtast nøgle</h1>
+            <p className='enter-key-text'>Indtast nøglen som de har modtaget ved skranken.</p>
```
Comment (TheMOHD3886) [inline src/Screens/VerifyKeyPage.jsx:31]: øhhm, det er faktisk "De" 🤓☝️

**Review by TheMOHD3886 — APPROVED** [review]
lgtm dawg 👍

**Review by Jatewo — CHANGES_REQUESTED** [review]
Ensure that cypress is not removed. Then I can probably approve.

**Review by TheMOHD3886 — APPROVED** [review]
double approve fordi du fjernede cypress 😛🙈💪

**Pull request closed without merging**